training/summary_utils.py

```diff
@@ -128,8 +128,7 @@ def add_overlay(self, tag, embed, img, alpha=0.8, cmap='inferno', add_ref=None):
         if len(img.shape) == 4:
             img = img[0]
         # Normalize the image.
-        img = img - img.min()
-        img = img/img.max()
+        img = img/255
```

Comment:
If I understand this correctly you're removing the normalization. You put the images in the 0-1 range but you don't set the mean to any particular value. Can you explain why you did that?

Reply:
The main idea is only to get back the original image. You just need to get it back from [0-255] to [0-1], the image won't be normalizes, but just like original jpeg image wasn't

Reply:
Maybe a better name would be "Remove image Normalization", then. Any reasons for that? The initial point of the normalization was to reduce some of the variation due to illumination.

Reply:
Sorry, I didn't realize this is applied to the summary images and not the training ones. If your confortable with it you can merge it.